ada-feeding-sim.rosinstall

```diff
@@ -0,0 +1,11 @@
+- git: {local-name: ada_demos, uri: 'https://github.com/personalrobotics/ada_demos.git'}
+- git: {local-name: ada_description, uri: 'https://github.com/personalrobotics/ada_description.git'}
+- git: {local-name: ada_launch, uri: 'https://github.com/personalrobotics/ada_launch.git'}
+- git: {local-name: aikido, uri: 'https://github.com/personalrobotics/aikido.git'}
+- git: {local-name: libada, uri: 'https://github.com/personalrobotics/libada.git'}
+- git: {local-name: pose_estimators, uri: 'https://github.com/personalrobotics/pose_estimators.git'}
+- git: {local-name: pr_assets, uri: 'https://github.com/personalrobotics/pr_assets.git'}
+- git: {local-name: pr_control_msgs, uri: 'https://github.com/personalrobotics/pr_control_msgs.git'}
+- git: {local-name: pr_ros_controllers, uri: 'https://github.com/personalrobotics/pr_ros_controllers.git'}
+- git: {local-name: pr_tsr, uri: 'https://github.com/personalrobotics/pr_tsr.git'}
+- git: {local-name: rewd_controllers, uri: 'https://github.com/personalrobotics/rewd_controllers.git'}
```

Comment:
Need to add the following repositories:
`bite_selection_package`
`food_detector`
`pytorch_retinanet`
(These are no longer submodules of food_detector)
I also want to add `sim_face_detector` to `face_detection` repository and make that a sim-dependency as well.
I can work on that tomorrow morning and let you know when it's ready. 

Why do we need rewd_controllers, pr_ros_controllers, and pr_control_msgs here?